docs/cli.md

```diff
@@ -51,10 +51,25 @@ you can run the following command from your `openwhisk` directory:
 ./bin/wsk property set --auth `cat ansible/files/auth.guest`
 ```
 
-**Tip:** The OpenWhisk CLI stores the properties set in `~/.wskprops` by default. The location of this file can be altered by setting the `WSK_CONFIG_FILE` environment variable.
+**Tip:** The OpenWhisk CLI stores properties in the `~/.wskprops` configuration file by default. The location of this file can be altered by setting the `WSK_CONFIG_FILE` environment variable.
+
+The required properties described above have the following keys in the `.wskprops` file:
+
+- **APIHOST** - Required key for the API host value.
+- **AUTH** - Required key for the Authorization key.
 
 To verify your CLI setup, try [creating and running an action](./samples.md).
 
+### Optional Whisk Properties
+
+Some OpenWhisk providers make use of optional properties that can be added to the `.wskprops` file.  The following keys are optional:
+
+- **NAMESPACE** - Optional, provider-relative identifier for a namespace to be used when managing OpenWhisk entities (e.g. Actions, Triggers, Rules, etc.) with the CLI.
```

Comment:
i dont think we need this at all - in fact i thought we removed recognition of this property from the cli.

Reply:
I will check... but i had a .wskprops created in July timeframe that added this key (although via the IBM plugin)

Reply:
NAMESPACE is still in the go-client... 
https://github.com/apache/openwhisk-client-go/blob/master/whisk/wskprops.go

Reply:
but gone/ignored by the cli itself https://github.com/apache/openwhisk-cli/pull/434/files

Reply:
it appears that the IBM plugin requires it and adds it back in if you delete it...  I will remove from the open docs.